src/main/java/com/example/spring/controller/BookController.java

```diff
@@ -50,8 +50,8 @@ public BookDtoWithoutCategoryIds findById(@PathVariable Long id) {
     @PreAuthorize("hasRole('ADMIN')")
     @PostMapping
     @Operation(summary = "Create a new book", description = "Create a new book")
-    public BookDto save(@RequestBody @Valid CreateBookRequestDto requestDto) {
-        return bookService.save(requestDto);
+    public void save(@RequestBody @Valid CreateBookRequestDto requestDto) {

```

Comment:
why did you decide to make it void?

Reply:
I think that delete method should return void

Reply:
It's not delete method, it's save method) 
Revert changes to return BookDto